test/blackbox-tests/test-cases/deprecated-library-name/redirect-cycle.t

```diff
@@ -0,0 +1,17 @@
+Creating an invalid (deprecated_libary_name ..) is an error
```

Comment:
What is the difference between this test and `invalid-redirect.t`?

Reply:
It was a botched renamed. Good catch.